### DISCUSSION TITLE
about file list order

### DISCUSSION THREAD

**Discussion by rookiez7** [wkentaro/labelme#1150] [Q&A / Help] [answered]
when i load a folder,the file in _File List_ is not ordered,how should i do fix it?
![5](https://user-images.githubusercontent.com/35830869/180188165-9af94105-ea8b-40ad-8607-22ea2c43d157.png)

**Comment by wkentaro** [comment, marked as answer]
this should be addressed by https://github.com/wkentaro/labelme/pull/990 and released at v5.0.0.
Can you report `labelme --version`?

**Reply by rookiez7** [reply]
Thanks,and now I have sloved this preblem by create a new python environment and install package,version`5.0.1`
About this preblem,first this was no preblem when i install the version `5.0.1`，then the problem arises when I changed version `5.0.1` to `4.5.6` and change it back to `5.0.1`.In the process，there is problem `4.5.6` and last `5.0.1`. 
Another preblem,when i turn on `auto_save`,It can't work,and the `save` function will also be disabled，how i slove it?that make me sue `ctrl+s` to save everytime

**Reply by wkentaro** [reply]
Can u open an issue for that? It might be a bug.

**Reply by rookiez7** [reply]
which one?`file list` or `auto_save`?

**Reply by wkentaro** [reply]
`file list` should be fine with v5.0.0, so please open an issue for `auto_save`.
